src/rt/backtrace/elf.d

```diff
@@ -11,16 +11,18 @@
 
 module rt.backtrace.elf;
 
-version(linux) version = linux_or_freebsd;
-else version(FreeBSD) version = linux_or_freebsd;
+version(linux) version = glibc_or_bsdlibc;
```

Comment:
This won't actually enable `Glibc`, as `linux` refers to kernel APIs.  I don't know where these declarations are from, so either change the version selected to `CRuntime_Glibc` or the new version to `linux_or_bsd`.